src/OpenTK/Platform/Windows/WinRawKeyboard.cs

```diff
@@ -134,19 +121,17 @@ public void RefreshDevices()
                     {
                         // This is a terminal services device, skip it.
                     }
-                    else if (dev.Type == RawInputDeviceType.KEYBOARD || dev.Type == RawInputDeviceType.HID)
+                    else if (dev.Type == RawInputDeviceType.Keyboard || dev.Type == RawInputDeviceType.Hid)
                     {
                         // This is a keyboard or USB keyboard device. In the latter case, discover if it really is a
                         // keyboard device by qeurying the registry.
                         var regkey = GetRegistryKey(name);
                         if (regkey == null)
-                        {
                             continue;
-                        }
 
-                        var deviceDesc = (string)regkey.GetValue("DeviceDesc");
-                        var deviceClass = (string)regkey.GetValue("Class");
-                        var deviceClassGUID =
+                        string deviceDesc = (string)regkey.GetValue("DeviceDesc");
```

Comment:
not ToString()? typename name = (typename) is by convention?

Reply:
GetValue returns an `object`. `ToString` does not perform type conversion.

Reply:
then why not `string GetValue(string)`?